examples/get_public_key.rs

```diff
@@ -0,0 +1,23 @@
+//
+// Copyright 2021 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use openapi::apis::{configuration::Configuration, pubkey_api};
+
+#[tokio::main]
+async fn main() {
+    let configuration = Configuration::default();
+    let pubkey = pubkey_api::get_public_key(&configuration, None).await;
+    println!("{:#?}", pubkey.unwrap());
```

Comment:
The formatting on this one could be tweaked to be a bit more readable, but not a big deal.